docs/en/Running-Hive-with-Alluxio.md

```diff
@@ -35,23 +35,26 @@ export HIVE_AUX_JARS_PATH={{site.ALLUXIO_CLIENT_JAR_PATH}}:${HIVE_AUX_JARS_PATH}
 
 Alternatively, advanced users can choose to compile this client jar from the source code. Follow the instructs [here](Building-Alluxio-Master-Branch.html#compute-framework-support) and use the generated jar at `{{site.ALLUXIO_CLIENT_JAR_PATH_BUILD}}` for the rest of this guide.
 
-## Create Hive Tables on Alluxio
+## Use Alluxio as One Option to Store Hive Tables
```

Comment:
Should this say "1. ...", and the other section say "2. ..."?

Reply:
thanks. that's what I want too. will fix